sphinx/source/users/deploy_app.rst

```diff
@@ -0,0 +1,28 @@
+Deploy an Application
+=====================
+
+The quickest way to deploy a CCF application is to use the `deploy_ccf.sh <https://github.com/microsoft/CCF/blob/master/deploy_ccf.sh>`_ script, specifying the :ref:`CCF application <Writing CCF Applications>` to run.
```

Comment:
I think this should say "enclave image" rather than "CCF application" - it takes a single compiled binary image, not some prior representation.